rules/windows/process_creation/win_vul_java_remote_debugging.yml

```diff
@@ -9,10 +9,10 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine: '*transport=dt_socket,address=*'
+        CommandLine|contains: 'transport=dt_socket,address='
     exclusion:
-        - CommandLine: '*address=127.0.0.1*'
-        - CommandLine: '*address=localhost*'
+        - CommandLine|contains: 'address=127.0.0.1'
+        - CommandLine|contains: 'address=localhost'
```

Comment:
it will not pass the test. you need to add '|all' to one of the lines, either 14 or 15

Reply:
nope, it will. sorry. it is `OR` in here